src/core/title.js

```diff
@@ -69,6 +75,19 @@ function setDocumentTitle(conf, h1Elem) {
     documentTitle = `Preview of PR #${conf.prNumber}: ${documentTitle}`;
   }
 
+  if (conf.hasOwnProperty("level")) {
+    if (validLevel(conf.level)) {
+      h1Elem.innerHTML = `${h1Elem.innerHTML} Level ${Number(conf.level)}`;
```

Comment:
Using .innerHTML makes me a bit uncomfortable. We are just appending  "Level ${conf.level}", so we can probably just `h1Elem.append()`. Also, we probably don't need to cast it into a Number, as it's just going to end up as a string.